app/db/index.js

```diff
@@ -0,0 +1,7 @@
+const Sequelize = require("sequelize");
+const { name, username, password, host } = require("../config").database;
+
+module.exports = new Sequelize(name, username, password, {
+  host,
+  dialect: "postgres"
```

Comment:
Would it be hard to make this configurable? Then we could be relatively database agnostic just in case.